controllers/ibmblockcsi_controller.go

```diff
@@ -112,8 +110,10 @@ func (r *IBMBlockCSIReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	reqLogger := log.WithValues("Request.Namespace", req.Namespace, "Request.Name", req.Name)
 	reqLogger.Info("Reconciling IBMBlockCSI")
 
+	r.ControllerHelper = common.NewControllerHelper(r.Client, log)
+
 	// Fetch the IBMBlockCSI instance
-	instance := ibmblockcsi.New(&csiv1.IBMBlockCSI{}, r.ServerVersion)
+	instance := controller_instance.New(&csiv1.IBMBlockCSI{}, r.ServerVersion)
```

Comment:
please consider changing the name of `controller_instance`, it's implying that it's already an instance of type controller.

Reply:
how about `customResource`?

Reply:
I think too general

Reply:
@oriyarde @ArbelNathan what do you think on `controllerWrapper`/ `costumResourceWrapper`

Reply:
why too general? maybe I misunderstood the meaning of this object.
I thought it exactly represents any custom resource of our operator.
I don't see how this is a `wrapper`

Reply:
is it more of a `CRUtils`?

Reply:
yes sound ok for me, @ArbelNathan?

Reply:
sure

Reply:
done